Makefile

```diff
@@ -0,0 +1,25 @@
+GOTEST := go test
+TAGS :=
+COVERAGE_FILE := coverage.out
+
+.PHONY: codespell
+codespell:
+	@echo "Running codespell"
+	@# codespell
+
+.PHONY: test
+test:
+	@echo "Runnning tests"
+	@# go test ./... -count=1
+
+.PHONY: testrace
+testrace:
+	@echo "Running tests with race flag"
+	@# go test ./... -count=1000 -race
+
+.PHONY: coveralls
+coveralls:
+	@echo "Running tests with coveralls"
+	@# go get github.com/mattn/goveralls
+	@# go install github.com/mattn/goveralls
+	@# goveralls -coverprofile=$(COVERAGE_FILE) -service=github
```

Comment:
Why are the lines commented? Please, add some empty files or configuration files to make it work.

Reply:
fixed